resources/user-agents/browsers/android-webview/android-webview-4-0.json

```diff
@@ -325,6 +325,7 @@
             "MTC SMART Run": "MTC SMART Run",
             "Oysters T72H 3G": "Oysters T72H 3G",
             "Oysters T72M 3G": "Oysters T72M 3G",
+            "P7-L10": "Huawei P7-L10",
```

Comment:
This device exists in the Android Browser 4.0 file, which caused one of the UAs in the tests to be detected as Android Browser instead of Android Webview (`get_browser` detected it as Webview, oddly enough).

>Mozilla/5.0 (Linux; Android 4.4.2; HUAWEI P7-L10 Build/HuaweiP7-L10) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/30.0.0.0 Mobile Safari/537.36